config/routes.rb

```diff
@@ -80,5 +80,9 @@
   match '/test/install_bot' => 'test#install_bot', via: :get
   match '/test/add_team_user' => 'test#add_team_user', via: :get
   match '/test/create_imported_standalone_fact_check' => 'test#create_imported_standalone_fact_check', via: :get
+  match '/test/create_saved_search' => 'test#create_saved_search', via: :get
```

Comment:
I didn't see this method in the `TestController`.

Reply:
thanks for catching that, @caiosba. I forgot to delete it. 
Done :saluting_face: